src/bench/wallet_migration.cpp

```diff
@@ -58,6 +63,11 @@ static void WalletMigration(benchmark::Bench& bench)
         mtx.vout.emplace_back(COIN, scripts_watch_only.at(j % NUM_WATCH_ONLY_ADDR));
         mtx.vin.resize(2);
         wallet->AddToWallet(MakeTransactionRef(mtx), TxStateInactive{}, /*update_wtx=*/nullptr, /*fFlushOnClose=*/false, /*rescanning_old_block=*/true);
+        {
+            WalletBatch batch(wallet->GetDatabase());
+            batch.WriteKey(pubkey, key.GetPrivKey(), CKeyMetadata());
```

Comment:
The empty metadata field smells bad. Double check migration.

Reply:
I've checked that keys (500 txs on local addresses) and scripts (20 wo imported addresses) are being migrated correctly (the first at `ApplyMigrationData` in `for (auto& desc_spkm : data.desc_spkms) {` and the second at `DoMigration` in `for (const auto& [desc_str, creation_time] : data->watch_descs) {`).

Reply:
I think empty metadata is fine, there isn't really much metadata to be had for imports and randomly generated keys. It's basically just the creation time.